lib/galaxy/tool_shed/galaxy_install/tools/tool_panel_manager.py

```diff
@@ -54,22 +54,33 @@ def add_to_shed_tool_config(self, shed_tool_conf_dict, elem_list):
             root = tree.getroot()
             for elem in root:
                 config_elems.append(elem)
+            # Build a list of tool panel section IDs in memory
```

Comment:
Why did you change this ? This looks like a more verbose version of what you had before.